src/modules/variables.js

```diff
@@ -0,0 +1,9 @@
+export const refreshDiv = document.querySelector('#refresh-img');
+export const enterDiv = document.querySelector('#enter-img');
+export const refreshImg = new Image();
+export const enterImg = new Image();
+export const toDoList = JSON.parse(localStorage.getItem('compiled'));
+export const listSort = (arr) => arr.slice().sort((a, b) => a.index - b.index);
+export const addInput = document.querySelector('.add-input');
+export const listWindow = document.querySelector('.list-window');
+export const clearAll = document.querySelector('#clear-done');
```

Comment:
Nicely done work, but I think you can improve your work readability if you actually write your functions here and export them to your index.js file

Reply:
Thank you Victor. I've made this changes in the same action as the first change request. The list functions have been moved here.